modules/_labs/drawer/react/spec/Drawer.spec.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react';
+import {render, fireEvent} from '@testing-library/react';
+
+import {Drawer, DrawerHeader, DrawerDirection} from '../index';
+
+describe('Drawer', () => {
+  test('should call a callback function', async () => {
+    const cb = jest.fn();
+    const {findByLabelText} = render(
+      <Drawer
+        header={
+          <DrawerHeader iconLabel={'Close'} title={'Header Title'} onClose={cb}></DrawerHeader>
```

Comment:
Is this a test of the `DrawerHeader` component? Is the `Drawer` component necessary?